19_shamshaw/readme.md

```diff
@@ -1,39 +1,54 @@
 # Occurence of Streamflow Drought Events in the Upper Colorado River
 
-Add some text here on data set and R packages used 
+This data visualization shows the occurence of streamflow drought events observed at USGS streamgages in the Upper Colorado River Basin. For the purposes of this visualization, a streamflow drought event is defined as the time period when streamflow drops below the 5th percentile of streamflows that have been observed on that date (utilizing a 30-day moving window). The source data for this plot is daily streamflow records downloaded from  [USGS NWIS](https://waterdata.usgs.gov/nwis/sw). 
 
-Run the targets pipeline to build the base visualizations. 
+## Data Preparation
 
-# Messaging
+To prepare the plot data file the following processing steps were performed:
 
-## Tweet 1
+ - Audit streamflow records to filter all available data to only data from streamgages where records between 1980 and 2020 have 5% or less missing data.
```

Comment:
Maybe add a note here saying "This resulted in # gage locations in the Upper Colorado"